automation_tools/satellite6/upgrade/capsule.py

```diff
@@ -0,0 +1,161 @@
+import os
+import sys
+import time
+
+from tools import reboot
+from automation_tools import setup_capsule_firewall
+from automation_tools.repository import enable_repos, disable_repos
+from automation_tools.satellite6.capsule import generate_capsule_certs
+from automation_tools.utils import distro_info, update_packages
+from fabric.api import env, execute, run
+from tasks import (
+    create_rhevm_instance,
+    delete_rhevm_instance,
+    sync_capsule_repos_to_upgrade
+)
+from tools import copy_ssh_key
```

Comment:
Actually alphabetically 'tools' lib should appear last and the reboot func has been merged in this line.